This patch works changes the biarch of PPC to the match other archs

diff -Naur gcc-4.4.0.orig/gcc/config/rs6000/t-linux64 gcc-4.4.0/gcc/config/rs6000/t-linux64
--- gcc-4.4.0.orig/gcc/config/rs6000/t-linux64	2007-09-27 12:56:06.000000000 -0700
+++ gcc-4.4.0/gcc/config/rs6000/t-linux64	2009-07-14 10:14:54.000000000 -0700
@@ -14,13 +14,13 @@
 # it doesn't tell anything about the 32bit libraries on those systems.  Set
 # MULTILIB_OSDIRNAMES according to what is found on the target.
 
-MULTILIB_OPTIONS        = m64/m32 msoft-float
+MULTILIB_OPTIONS        = m64/m32
 MULTILIB_DIRNAMES       = 64 32 nof
 MULTILIB_EXTRA_OPTS     = fPIC mstrict-align
-MULTILIB_EXCEPTIONS     = m64/msoft-float
-MULTILIB_EXCLUSIONS     = m64/!m32/msoft-float
-MULTILIB_OSDIRNAMES	= ../lib64 $(if $(wildcard $(shell echo $(SYSTEM_HEADER_DIR))/../../usr/lib32),../lib32,../lib) nof
-MULTILIB_MATCHES        = $(MULTILIB_MATCHES_FLOAT)
+MULTILIB_EXCEPTIONS     =
+MULTILIB_EXCLUSIONS     =
+MULTILIB_OSDIRNAMES	= ../lib64 $(if $(wildcard $(shell echo $(SYSTEM_HEADER_DIR))/../../usr/lib32),../lib32,../lib)
+MULTILIB_MATCHES        = 
 
 softfp_wrap_start := '\#ifndef __powerpc64__'
 softfp_wrap_end := '\#endif'
